iceoryx_binding_c/source/c_server.cpp

```diff
@@ -75,19 +75,20 @@ iox_server_t iox_server_init(iox_server_storage_t* self,
         serverOptions.clientTooSlowPolicy = c2cpp::consumerTooSlowPolicy(options->clientTooSlowPolicy);
     }
 
-    new (self) UntypedServer(ServiceDescription{IdString_t(TruncateToCapacity, service),
-                                                IdString_t(TruncateToCapacity, instance),
-                                                IdString_t(TruncateToCapacity, event)},
-                             serverOptions);
+    auto* me = new UntypedServer(ServiceDescription{IdString_t(TruncateToCapacity, service),
```

Comment:
`auto*` is technically redundant and `auto` could be used. I know it adds some implicit compile check whether we really get a pointer, that may be the reason to use it but ...well it is a `new` call.

Reply:
I guess this is because of a clang-tidy warning ;)